examples/search_examples/public/components/app.tsx

```diff
@@ -143,6 +165,9 @@ export const SearchExamplesApp = ({
     const searchSubscription$ = data.search
       .search(request, {
         strategy,
+        sessionId: searchSessionId,
+        isStored: data.search.session.isStored(),
```

Comment:
As we discussed, let's not pass this explicitly for now and infer inside session service from `sessionId`? 
https://github.com/elastic/kibana/pull/83073/files#diff-3c08f8d5aa2d2ab21c4d43c3c199223026176fd4a470965b9f68c21e6cb5056b